rest-client-reactive-quickstart/src/main/java/org/acme/rest/client/Country.java

```diff
@@ -0,0 +1,20 @@
+package org.acme.rest.client;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Isn't that the default? @geoand

Reply:
Good point! I had completely forgot about this enhancement!

Reply:
done